### ISSUE TITLE
NRE in 1.8 

### ISSUE THREAD

**Issue by IceRaptor** [BattletechModders/LowVisibility#25]
Error code from 1.8 alpha has this stack trace:

```
Uploading Crash Report
NullReferenceException: Object reference not set to an instance of an object
  at LowVisibility.Patch.TurnDirector_OnCombatGameDestroyed.Postfix (BattleTech.TurnDirector __instance) [0x00029] in <a3d3375cb96f4533b9bb8b0f71901432>:0 
  at (wrapper dynamic-method) BattleTech.TurnDirector.OnCombatGameDestroyed_Patch3(object)
  at (wrapper dynamic-method) BattleTech.CombatGameState.OnCombatGameDestroyed_Patch9(object)
  at BattleTech.GameInstance.ClearCombat () [0x0000f] in <e1836ad313454eb4828ce697e61d02e0>:0 
  at (wrapper dynamic-method) BattleTech.UI.AAR_SalvageScreen.OnCompleted_Patch1(object)
  at BattleTech.UI.AAR_SalvageScreen.ReceiveButtonPress (System.String button) [0x0000d] in <e1836ad313454eb4828ce697e61d02e0>:0 
  at UnityEngine.Events.InvokableCall`1[T1].Invoke (T1 args0) [0x00011] in <9a6ceb7f013a4a599806f492fc789483>:0 
  at UnityEngine.Events.CachedInvokableCall`1[T].Invoke (System.Object[] args) [0x00001] in <9a6ceb7f013a4a599806f492fc789483>:0 
  at UnityEngine.Events.UnityEvent.Invoke () [0x0006e] in <9a6ceb7f013a4a599806f492fc789483>:0 
  at BattleTech.UI.HBSButton.OnPointerUp (UnityEngine.EventSystems.PointerEventData eventData) [0x0006d] in <e1836ad313454eb4828ce697e61d02e0>:0 
  at UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerUpHandler handler, UnityEngine.EventSystems.BaseEventData eventData) [0x00008] in <689405fdf241479fa6250af7e4b0368a>:0 
  at UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) [0x00070] in <689405fdf241479fa6250af7e4b0368a>:0 
UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
UnityEngine.DebugLogHandler:LogException(Exception, Object)
```

**Comment by IceRaptor** [comment]
```
Uploading Crash Report
NullReferenceException: Object reference not set to an instance of an object
  at BattleTech.TurnDirector.Update () [0x0000e] in <e1836ad313454eb4828ce697e61d02e0>:0 
  at BattleTech.CombatGameState.Update () [0x00021] in <e1836ad313454eb4828ce697e61d02e0>:0 
  at BattleTech.GameInstance.Update (System.Single deltaTime) [0x0000e] in <e1836ad313454eb4828ce697e61d02e0>:0 
  at BattleTech.UnityGameInstance.Update () [0x00013] in <e1836ad313454eb4828ce697e61d02e0>:0 
 
(Filename: <e1836ad313454eb4828ce697e61d02e0> Line: 0)
```

**Comment by IceRaptor** [comment]
Fixed in 0.6.2
